src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -64,6 +65,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
+            ++m_version;
```

Comment:
> But don't get me wrong, I totally understand that we don't really need the correct value in the field for compatibility with netfx. I'm just not sure if that's also true for earlier versions or other frameworks like mono.

It's very strange to me that we're caring about the value we set it to and then never reading it.  If we were using it in the enumerator, I'd understand, but this seems to be doing work for nothing.